app/templates/_instructions.html

```diff
@@ -0,0 +1,174 @@
+{% macro r(dataset) -%}
+<p>In order to use Data Package in R follow instructions below:</p>
+<div class="highlight">
+<pre>
+<span></span>install.packages<span class="p">(</span><span class="s">"devtools"</span><span class="p">)</span>
+<span class="kn">library</span><span class="p">(</span>devtools<span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"hadley/readr"</span><span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"ropenscilabs/jsonvalidate"</span><span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"ropenscilabs/datapkg"</span><span class="p">)</span>
+
+<span class="c1">#Load client</span>
+<span class="kn">library</span><span class="p">(</span>datapkg<span class="p">)</span>
+
+<span class="c1">#Get Data Package</span>
+datapackage <span class="o">&amp;</span>lt<span class="p">;</span>
+<span class="o">-</span> datapkg_read<span class="p">(</span>
+<span class="s">"https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest"</span><span class="p">)</span>
+
+<span class="c1">#Package info</span>
+<span class="kp">print</span><span class="p">(</span>datapackage<span class="p">)</span>
+
+<span class="c1">#Open actual data in RStudio Viewer</span>
```

Comment:
there are no space between comment and code snippets except `#Open actual data in RStudio Viewer ` and `View(datapackage$data$"demo-resource")`

![image](https://cloud.githubusercontent.com/assets/15143785/25886747/13a1db40-3570-11e7-8eeb-28f145273b00.png)

Lets stay consistent and either add for all of them or remove (prob remove)